gpcontrib/gg_tables_tracking/isolation2/sql/recovery_failover.sql

```diff
@@ -0,0 +1,110 @@
+-- This test triggers failover of content 1 and checks
+-- the correct tracking state behaviour after recovery
+!\retcode gpconfig -c shared_preload_libraries -v 'gg_tables_tracking';
```

Comment:
Don't touch other extensions in `shared_preload_libraries`!
For example
```suggestion
!\retcode gpconfig -c shared_preload_libraries -v "$(psql -At -c "SELECT array_to_string(array_append(string_to_array(current_setting('shared_preload_libraries'), ','), 'gg_tables_tracking'), ',')" postgres)"
```